files/galaxy/config/user_preferences_extra_conf.yml

```diff
@@ -1,4 +1,45 @@
 preferences:
+    # the key you can refer to
+    apollo_url_01:
```

Comment:
this has never been used fwiw

Reply:
So, can be removed?
I think the user preferences panel is too busy

Reply:
yep! (And I agree!)

Reply:
It was in there as a demo of the user prefs, but, I was never going to let apollo users just store their passwords in clear. maybe now with the new 'vault' it'll be an idea, because we'd love to use the feature, but until then please get rid of it, it's annoying